lib/service/service.go

```diff
@@ -135,6 +135,12 @@ func (process *TeleportProcess) connectToAuthService(role teleport.Role) (*Conne
 	storage := utils.NewFileAddrStorage(
 		filepath.Join(process.Config.DataDir, "authservers.json"))
 
+	if len(process.Config.AuthServers) > 0 {
```

Comment:
This will overwrite the previous value if it exists, so we need `SetIfNotExists` semantics